src/Model/Todo/Todo.php

```diff
@@ -231,6 +253,16 @@ protected function whenDeadlineWasAddedToTodo(DeadlineWasAddedToTodo $event)
     protected function whenReminderWasAddedToTodo(ReminderWasAddedToTodo $event)
     {
         $this->reminder = $event->reminder();
+        $this->reminded = false;
+    }
+
+    /**
+     * @param TodoAssigneeWasReminded $event
+     * @return void
+     */
+    protected function whenTodoAssigneeWasReminded(TodoAssigneeWasReminded $event)
```

Comment:
Yeah but not about the same reminder. This is what is prevented here. The assignee should only be notified again if the reminder changes, therefore I set $this->reminded to false during whenReminderWasAddedToTodo